.github/workflows/tutorial-tests.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         python -m venv venv
         source venv/bin/activate
         pip install requests
-        pip install duckdb
+        pip install duckdb==1.0.0
```

Comment:
Can you also update the instructions in the [README.md](https://github.com/unitycatalog/unitycatalog/blob/main/README.md#operate-on-delta-tables-with-duckdb) to pin `duckdb==1.0.0` (vs. the latest)?

Reply:
Done (the readme just said in words to use >=1.0.0, and I changed it to specify 1.0.0).